src/bin/ion/commands/generate/templates/java/sequence.templ

```diff
@@ -17,7 +17,7 @@ import java.io.IOException;
 class {{ model.name }} {
     private java.util.ArrayList<{{ sequence_info["element_type"] | fully_qualified_type_name }}> value;
 
-    public {{ model.name }}() {}
+    private {{ model.name }}() {}
 
     public java.util.ArrayList<{{ sequence_info["element_type"] | fully_qualified_type_name }}> getValue() {
```

Comment:
I'm sorry I missed this earlier. When working with Java collections, you should use the widest allowable interface for the API (so `List` in this case) and use the specific implementation only when you're actually constructing a new instance.

Reply:
BTW, It's fine if you want to follow up on this in another PR.